cardano-db-sync/test/Cardano/DbSync/Gen.hs

```diff
@@ -106,11 +107,11 @@ syncNodeConfig loggingCfg =
 syncInsertConfig :: Gen SyncInsertConfig
 syncInsertConfig =
   Gen.choice
-    [ pure FullInsertOptions
-    , pure OnlyUTxOInsertOptions
-    , pure OnlyGovInsertOptions
-    , pure DisableAllInsertOptions
-    , SyncInsertConfig <$> syncInsertOptions
+    [ pure $ SyncInsertConfig (Just "full") fullInsertOptions
+    , pure $ SyncInsertConfig (Just "only_utxo") onlyUTxOInsertOptions
+    , pure $ SyncInsertConfig (Just "only_gov") onlyGovInsertOptions
+    , pure $ SyncInsertConfig (Just "disable_all") disableAllInsertOptions
+    , SyncInsertConfig Nothing <$> syncInsertOptions
```

Comment:
We might want to add another choice in here to generate a preset AND overrides

Reply:
I think we're still missing some cases, because of `parseOverrides`, but I can't think of a good way to deal with this.